### PR TITLE
be able to wait until codebuild build is done

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: src/modules/0.0.22/aws_codebuild/entity/build.ts [hunk 1]
```diff
@@ -73,4 +73,11 @@ export class CodebuildBuildImport {
     name: 'project_name',
   })
   project: CodebuildProject;
+
```
Comment (yrobla) [inline src/modules/0.0.22/aws_codebuild/entity/build.ts:76]: why not always wait? that is the default behaviour for other modules...

**Review by aguillenv — APPROVED** [review]

File: src/modules/0.0.22/aws_codebuild/entity/build.ts (hunk 1 above)
Comment (yrobla) [reply src/modules/0.0.22/aws_codebuild/entity/build.ts:76]: in the acm request certificate, i made it wait until the cert was valid. I also did the same for codedeploy, and now with codepipeline...

Comment (aguillenv) [reply src/modules/0.0.22/aws_codebuild/entity/build.ts:76]: that's true, rds also takes a lot of time and we wait for it

Comment (depombo) [reply src/modules/0.0.22/aws_codebuild/entity/build.ts:76]: will change the default behavior!

**Review by dfellis — APPROVED** [review]

**Review by aguillenv — APPROVED** [review]

**Review by yrobla — APPROVED** [review]

**Pull request merged**